### ISSUE TITLE
test(e2e): basic geometric twap test

### ISSUE THREAD

**Issue by p0mvn** [osmosis-labs/osmosis#3728]
## Background

It would be nice to have a basic twap e2e test to ensure that the API works as expected and that the migrations are done correctly.

## Suggested Design

1. Create a pool with `1_000_000` `uosmo`, pool 2 with `2_000_000` `uatom`.

2. Query geometric twap. Base = uosmo, result = 2

3. Swap `1_000_000` `uatom` for `uosmo`.

4. Query geometric twap to now with start time after swap. Base = uosmo. Assert the result < 2

## Acceptance Criteria
- e2e test implemented per suggestion
- passes

**Comment by p0mvn** [comment]
Blocked on #3514
